ui/app/components/config-pki-ca.js

```diff
@@ -140,15 +140,10 @@ export default Component.extend({
       const isUpload = this.model.uploadPemBundle;
       model
         .save({ adapterOptions: { method } })
-        .then((m) => {
+        .then(() => {
           if (method === 'setSignedIntermediate' || isUpload) {
             this.send('refresh');
             this.flashMessages.success('The certificate for this backend has been updated.');
-          } else if (!m.get('certificate') && !m.get('csr')) {
```

Comment:
According to the backend team, a certificate will always be generated so no need for this `else if`

Reply:
Obviously, pending any actual errors. :-) But if its a valid request, you will always get a cert or CSR by design now.